docs/data-model.md

```diff
@@ -0,0 +1,78 @@
+| A batch PR is submitted for this and other terms (for import and creation)                                                                                                   | awaiting_import      | reviewed                                                        | awaiting_import                                                  |
+| A term import is completed — the new ontology term status is caught automatically during a periodic OLS status refresh. <br>After import the same term will now be "current" | current              | reviewed                                                        | current                                                          |
+|                                                                                                                                                                              |                      |                                                                 |                                                                  |
+| **Case 3. Curator chooses a new term which is not in _any_ ontology (needs creation)**                                                                                       | needs_creation       | not_reviewed                                                    | awaiting_review                                                  |
+| necessary number of reviews is accumulated                                                                                                                                   | needs_creation       | reviewed                                                        | needs_creation                                                   |
+| A batch PR is submitted for this and other terms (for import and creation)                                                                                                   | awaiting_creation    | reviewed                                                        | awaiting_creation                                                |
+| After the PR is closed and new terms are created, the curator <br>manually selects a new term                                                                                | current              | not_reviewed                                                    | awaiting_review                                                  |
+| A necessary number of reviews for the newly created term is accumulated                                                                                                      | current              | reviewed                                                        | current                                                          |
+
+### Diagram:
+
+![](https://i.imgur.com/tUBBipu.png)
+
+Mermaid diagram source code :
```

Comment:
It's best to move this into a separate file, like `docs/statuses_diagram.mmd` (`.mmd` is [the official extension](https://github.com/mermaid-js/mermaid/issues/147) for Mermaid source code).